File: week-06-Python/sentimental/cash/cash.py
from cs50 import get_float

change = -1

while change < 0:
    change = 100 * (get_float("Change: "))

# print(change)
coins = 0


def get_change(change, coin):
    coins = 0
    while change >= coin:
        coins += (int)(change / coin)
        change = (int)(change % coin)
    return coins, change


for i in [25, 10, 5, 1]:
    new_coins, change = get_change(change, i)
    coins += new_coins

print(coins)

File: week-01-C/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    int change;
    do{
        change = get_int("Change owed: ");
        // printf("%i", change);
    }
    while(change<0);

    int coins=0;
    while(change>0)
    {
        if(change>=25)
        {
            coins += change/25;
            change = change%25;
        }
        else if(change<=25 && change>=10)
        {
            coins += change/10;
            change = change%10;
        }
        else if(change<=10 && change>=5)
        {
            coins += change/5;
            change = change%5;
        }
        else
        {
            coins += change;
            change=0;
        }
    }
    printf("%i\n", coins);

}
